Observation/Observation-MultipleTumors-Example.json

```diff
@@ -0,0 +1,48 @@
+{
+  "resourceType": "Observation",
+  "id": "Observation-MultipleTumors-Example",
+  "status": "final",
+  "code": {
+    "coding": [
+      {
+        "system": "http://snomed.info/sct",
+        "code": "128462008",
+        "display": "Metastatic malignant neoplasm (disorder)"
+      }
+    ]
+  },
+  "subject": {
+    "reference": "Patient/Patient-ZelmaHadjkiss-Example",
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449307555"
+    }
+  },
+  "specimen": {
+    "identifier": {
+      "system": "https://fhir.add.nhs.uk/Id/specimenId",
+      "value": "RA257630"
+    }
+  },
+  "valueQuantity": {
+    "value": 25,
+    "unit": "%",
+    "system": "http://unitsofmeasure.org",
+    "code": "%"
+  },
+  "component": [
+    {
+      "code": {
+        "coding": [
+          {
+            "system": "http://snomed.info/sct",
+            "code": "369755005",
+            "display": "Multiple tumors (qualifier value)"
```

Comment:
Multiple tumours should now be recorded as multiple bodysite references as per the extension on UKCore Specimen.collection.bodysite ([https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-BodySiteReference](https://simplifier.net/resolve?scope=HL7FHIRUKCoreR4@current&filepath=structuredefinitions/Extension-UKCore-BodySiteReference.xml)) so remove the component and attempt to add a BodySite resource